docs/add-non-um-users.html

```diff
@@ -3,5 +3,34 @@
 title: Add Non-UM Users
 ---
 
-<h1 class="mt-5">Add Non-UM Users Documentation</h1>
-<p>Follow documentation for this Feature</p>
\ No newline at end of file
+<h1 class="mt-5">Add Non-UM Users</h1>
+<p>External users (those without a U-M uniqname) can be added Canvas courses with the Course Manager tool. New external users who have never accessed Canvas will receive information by email to help them select a login method.  by course managers and administrators.</p>
+<h5>Who Can Add Non-UM Users?</h5>
+<ul class="mx-3">
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers who normally have access to the People tool</li>
+</ul>
+<h5>How to Add Non-UM Users?</h5>
+<p>External users can be added one at time by typing in the user’s email address, or in bulk using a CSV file.</p>
+<h5>Adding External Users One at a Time</h5>
```

Comment:
```suggestion
<h2>Adding External Users One at a Time</h2>
```

Reply:
I'd make this an `h6`, since that's what the level above is.